### ISSUE TITLE
Dependency with numeric GitHub owner produces invalid JavaScript variable name

### ISSUE THREAD

**Issue by BurntCaramel** [elm/compiler#1868]
Example: https://ellie-app.com/48NmnW2HchVa1

When a dependency has a GitHub owner that starts with a number, e.g. `1602/json-value`:

```json
"dependencies": {
        "direct": {
            "1602/json-value": "3.0.1",
…
```

Then the produced JavaScript variable that is based on that GitHub owner name starts with a number too, which is not valid for JavaScript. 

```js
…

var 1602$json_value$Json$Value$BoolValue = function (a) {
	return {$: 'BoolValue', a: a};
};

…
```

**Comment by lydell** [comment]
Related: #1763

**Comment by evancz** [comment]
Using the development version of the compiler, the names are now getting generated in a format that'll work for user names that start with numbers: `$1602$json_value$Json$Value$BoolValue`. So when I compile your SSCCE, it shows up and runs.

So the fix should become available with the upcoming 0.19.1 release of Elm.
